### DISCUSSION TITLE
qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "" even though it was found.

### DISCUSSION THREAD

**Discussion by myj8888** [CommonRoad/commonroad-scenario-designer#26] [Q&A] [answered]
before“pip install commonroad-scenario-design”， I process code_example.py ,it is no error.but,after I "pip install commonroad-scenario-designeer",there are many problems,liking this picture.I hoope someone can help me,thanks.
<img width="640" alt="problem" src="https://github.com/user-attachments/assets/2fc85758-47b5-449a-8f9e-378b52edd978">

**Comment by smaierhofer** [comment, marked as answer]
are you using the CommonRoad Scenario Designer or commonroad-io?
The error is related to your system but I'm quite sure there exist many solutions out there, e.g., on stackoverflow.

**Reply by myj8888** [reply]
I've solved it based on the error message.thanks
